indico/htdocs/sass/base/_layout.scss

```diff
@@ -136,7 +136,14 @@ body > * {
     }
 }
 
-/* Flexbox layout */
+
+// -- Flexbox layout -----------------------------------------------------------
```

Comment:
I'd keep the simple `/* */` I agree it's pretty ugly having an asymmetric line of dashes